core/src/main/java/hudson/model/ListView.java

```diff
@@ -481,6 +490,18 @@ public void setIncludeRegex(String includeRegex) {
             this.includePattern = Pattern.compile(includeRegex);
     }
 
+    @DataBoundSetter
+    public void setJobNames(List<String> jobNames) {
+        synchronized (this) {
```

Comment:
You could just make the method synchronized to do the same thing here. ;)

Reply:
Done